pages/design_documentation/ocf_io_engines.md

```diff
@@ -0,0 +1,292 @@
+---
+title: Cache modes implementation - I/O engines.
+last_updated: August 17, 2022
+toc: false
+permalink: ocf_io_engines.html
+---
+
+Open CAS implements several [**cache modes**](/cache_configuration.html), 
+each suitable for a different usage scenario. To achieve a desired mode of 
+operation, Open CAS Framework uses basic building blocks called **IO engines**. 
+These blocks implement specific algorithms, e.g. write-through or write-back 
+and handle special or corner cases as well.
```

Comment:
After iterating over example algorithms there should be second coma (to close the interruption in sentence).
Second, the ending of this sentence is not quite clear for me. Lets discuss it.

Reply:
I agree,  the last part of this sentence is not useful at all in this context. Fixed.